.claude/hooks/PreToolUse/invoke_false_completion_gate.py

```
@@ -0,0 +1,308 @@
+
+    Args:
+        is_pr_create: True if this is a `gh pr create` command, False for `git commit`.
+                      For commits, checks staged changes (git diff --cached).
+                      For PRs, checks branch diff against the merge base.
+    """
+    try:
+        if is_pr_create:
+            # For PR create, get the branch diff against the merge base
+            # First, find the merge base with the default branch
+            base_result = subprocess.run(
+                ["git", "rev-parse", "--abbrev-ref", "HEAD"],
```

Ruff (0.15.11)

[error] Starting a process with a partial executable path

(S607)